v2/pkg/cli/executor.go

```diff
@@ -180,7 +182,7 @@ func NewMirrorCmd(log clog.PluggableLoggerInterface) *cobra.Command {
 	cmd.AddCommand(NewPrepareCommand(log))
 	cmd.PersistentFlags().StringVarP(&opts.Global.ConfigPath, "config", "c", "", "Path to imageset configuration file")
 	cmd.Flags().StringVar(&opts.Global.LogLevel, "loglevel", "info", "Log level one of (info, debug, trace, error)")
-	cmd.Flags().StringVar(&opts.Global.WorkingDir, "dir", workingDir, "Assets directory")
+	cmd.Flags().StringVar(&opts.Global.WorkingDir, "with", "", "local storage directory for mirror to mirror workflow")
```

Comment:
In our domain, source is a container registry and destination is also a container registry. In this context here we are talking about the working-dir which is a different thing. We should be very careful with the words we use otherwise the customer experience will be very confusing. 

I would suggest to use **workspace** everytime we talk about the workingDir, because this is what it is, it is the oc-mirror workspace.